app/helpers/notifier_helper.rb

```diff
@@ -21,4 +30,13 @@ def comment_message(comment, opts={})
       I18n.translate "notifier.a_limited_post_comment"
     end
   end
+
+  def truncated_comment_message(comment)
+    if comment.post.public?
+      plain_text = comment.message.plain_text_without_markdown
+      truncate(plain_text, length: 180)
+    else
+      I18n.t "notifier.a_limited_post_comment"
+    end
+  end
```

Comment:
Same as above, this isn't needed, and #8035 should already use `comment.message.title` directly, so you don't need to do anything here.

Reply:
Removed as suggested